src/types.ts

```diff
@@ -19,4 +19,9 @@ export interface ImageRenderOptions extends RenderOptions {
   width: string | number;
   height: string | number;
   deviceScaleFactor?: string | number;
+
+  // Runtime options derived from the input
+  fullPageImage?: boolean;
+  scrollDelay?: number;
+  scaleImage?: number;
```

Comment:
I assume that's an easy way to make it work without changing the API version? @ryantxu

Reply:
I think reusing `deviceScaleFactor` is ok because:
1. the external param for this is already called "scale"! :)
2. would never want deviceScaleFactor != 1 and scale !=1 at the same time

In general, I don't really understand  why `deviceScaleFactor` exists -- i *think* you get identical results just scaling the image width/height.